zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -563,6 +563,10 @@ ascan.toolbar.requests.label   = Num requests:
 ascan.toolbar.progress.label   = Progress:
 ascan.toolbar.progress.select  = --Select Scan--
 ascan.url.popup                = Active Scan single URL
+ascan.panel.tab.scannedMessages   = Scanned Messages
```

Comment:
`Messages Sent`?

Reply:
hmm ok

Reply:
I'm suggesting "sent" instead of "scanned" as the latter could be interpreted as the original scanned messages.

Reply:
ok so it will be names as `Sent Messages` ?

Reply:
Yeah.